### PR TITLE
Release 2.18.1

### PR DESCRIPTION
**Pull request by michaelfig**
## Packages that have NEWS.md updates

```diff
--- a/packages/SwingSet/CHANGELOG.md
+++ b/packages/SwingSet/CHANGELOG.md
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [0.17.1](https://github.com/Agoric/agoric-sdk/compare/@agoric/swingset-vat@0.17.0...@agoric/swingset-vat@0.17.1) (2021-05-05)
+
+
+### Bug Fixes
+
+* **swingset:** force vattp to run on worker=local for now ([a6aff0a](https://github.com/Agoric/agoric-sdk/commit/a6aff0ac52de6ecd12b9b1c5c82958f502b549b3)), closes [#3039](https://github.com/Agoric/agoric-sdk/issues/3039)
+
+
+
+
+
 # [0.17.0](https://github.com/Agoric/agoric-sdk/compare/@agoric/swingset-vat@0.16.4...@agoric/swingset-vat@0.17.0) (2021-05-05)
 
 
--- a/packages/deployment/CHANGELOG.md
+++ b/packages/deployment/CHANGELOG.md
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [1.27.1](https://github.com/Agoric/agoric-sdk/compare/@agoric/deployment@1.27.0...@agoric/deployment@1.27.1) (2021-05-05)
+
+
+### Bug Fixes
+
+* cope with getting moddable submodule from agoric-labs ([a1a2693](https://github.com/Agoric/agoric-sdk/commit/a1a26931d17ade84ae97aa3a9d0e7c5c58a74491))
+
+
+
+
+
 # [1.27.0](https://github.com/Agoric/agoric-sdk/compare/@agoric/deployment@1.26.5...@agoric/deployment@1.27.0) (2021-05-05)
 
 
--- a/packages/xsnap/CHANGELOG.md
+++ b/packages/xsnap/CHANGELOG.md
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [0.6.1](https://github.com/Agoric/agoric-sdk/compare/@agoric/xsnap@0.6.0...@agoric/xsnap@0.6.1) (2021-05-05)
+
+
+### Bug Fixes
+
+* cope with getting moddable submodule from agoric-labs ([a1a2693](https://github.com/Agoric/agoric-sdk/commit/a1a26931d17ade84ae97aa3a9d0e7c5c58a74491))
+
+
+
+
+
 # [0.6.0](https://github.com/Agoric/agoric-sdk/compare/@agoric/xsnap@0.5.5...@agoric/xsnap@0.6.0) (2021-05-05)
 
 
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,18 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [2.18.1](https://github.com/Agoric/agoric-sdk/compare/@agoric/sdk@2.18.0...@agoric/sdk@2.18.1) (2021-05-05)
+
+
+### Bug Fixes
+
+* **swingset:** force vattp to run on worker=local for now ([a6aff0a](https://github.com/Agoric/agoric-sdk/commit/a6aff0ac52de6ecd12b9b1c5c82958f502b549b3)), closes [#3039](https://github.com/Agoric/agoric-sdk/issues/3039)
```
Comment (warner) [inline CHANGELOG.md:11]: BTW it doesn't close 3039, it's just a quick hopeful hack.

File: packages/SwingSet/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,17 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [0.17.1](https://github.com/Agoric/agoric-sdk/compare/@agoric/swingset-vat@0.17.0...@agoric/swingset-vat@0.17.1) (2021-05-05)
+
+
+### Bug Fixes
+
+* **swingset:** force vattp to run on worker=local for now ([a6aff0a](https://github.com/Agoric/agoric-sdk/commit/a6aff0ac52de6ecd12b9b1c5c82958f502b549b3)), closes [#3039](https://github.com/Agoric/agoric-sdk/issues/3039)
```
Comment (warner) [inline packages/SwingSet/CHANGELOG.md:11]: ditto.. does the changelog generator treat `refs #NN` the same as `closes #NN`?

**Review by dckc — APPROVED** [review]
nice work!

I should have communicated better about the moddable submodule change. I forgot about the non-git mode of build.js.

File: packages/SwingSet/CHANGELOG.md (hunk 2 above)
Comment (michaelfig) [reply packages/SwingSet/CHANGELOG.md:11]: > ditto.. does the changelog generator treat `refs #NN` the same as `closes #NN`?

It seems to.  I'd report a bug when I have time.

**Pull request merged**